h2o-algos/src/main/java/hex/word2vec/Word2VecModel.java

```diff
@@ -136,7 +136,7 @@ public void map(Chunk[] cs, NewChunk[] ncs) {
           }
         }
         offset = 0;
-      } while (seqOpen && ((chk = chk.nextChunk()) != null));
+      } while ((chk = chk.nextChunk()) != null);
```

Comment:
This is what fixes the issue. Even though we reached an end of the chunk and the sequence is not open (was closed by NA terminator) we will continue to the next chunk until we found the next NA (otherwise the beginning of the next chunk would be skipped when the map() is applied to the chunk).